media/css/cms/flare-template.css

```diff
@@ -479,3 +255,69 @@
     }
 
 }
+
+/* Styles used by Download Firefox Button messages */
+
+.fl-fx-download-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: calc(var(--scale-16) * 1px);
+}
+
+/* Hide the unsupported message by default */
+.fx-unsupported-message {
+    display: none;
+    text-align: center;
+    margin-bottom: 8px;
+}
+
+.fx-unsupported-message p {
+    margin-bottom: 8px;
+}
+
+/* Show the Windows unsupported message if on Windows with unsupported version for Firefox */
+.windows.fx-unsupported .fx-unsupported-message.win {
```

Comment:
Download buttons styles should be broken out into a separate file and then we need to make sure we're not breaking anything.